tests/__init__.py

```diff
@@ -12,30 +12,28 @@ def __init__(self, name, *args, **kwargs):
 
         
 class TestProjectManagement(TestCase):
-    root_path = abspath(dirname(dirname(__file__)))
+    root_path = dirname(dirname(__file__))
     etc_config = join(root_path, 'etc', 'example_project_management.yaml')
     root_test_path = join(root_path, 'tests')
     assets_path = join(root_test_path, 'assets')
     assets_deletion = join(assets_path, 'data_deletion')
+    config_file = 'example_project_management.yaml'
```

Comment:
Test classes can now define a field called `config_file`, which will be loaded from `etc` in `setUpClass`, instead of having to manually load a config in every test script.